external-crates/move/tools/move-package/src/compilation/model_builder.rs

```diff
@@ -85,14 +90,14 @@ impl ModelBuilder {
                         paths.into_iter().partition(|t| t.contains(filter));
                     if !true_targets.is_empty() {
                         new_targets.push(PackagePaths {
-                            name,
+                            name: name.clone(),
                             paths: true_targets,
                             named_address_map: named_address_map.clone(),
                         })
                     }
                     if !false_targets.is_empty() {
                         new_deps.push(PackagePaths {
-                            name,
+                            name: name.clone(),
```

Comment:
...how did this work before?

Reply:
It was just a `Symbol` before, which implements `Copy`